cpp/benchmarks/common/generate_input.cu

```diff
@@ -655,6 +717,23 @@ std::unique_ptr<cudf::table> create_sequence_table(std::vector<cudf::type_id> co
   return std::make_unique<cudf::table>(std::move(columns));
 }
 
+std::pair<rmm::device_buffer, cudf::size_type> create_random_null_mask(cudf::size_type size,
+                                                                       float null_probability,
+                                                                       unsigned seed)
+{
+  if (null_probability < 0.0f) {
+    return {rmm::device_buffer{}, 0};
```

Comment:
Why support this at all? Why not just require a positive float value? For that matter, why not just error check that 0 <= null_probability <= 1?

Reply:
used `std::optional` and added error check.